src/registry/artifactory.rs

```diff
@@ -143,12 +154,6 @@ impl Registry for Artifactory {
             "Server response has incorrect mime type: {content_type:?}"
```

Comment:
If we have an invalid mimetype, can we print debug information in the eyre stacktrace?

Currently the mimetype error is a little unhelpful + we should only verify the mimetype when the response code is exactly 200!